muni_portal/urls.py

```diff
@@ -9,14 +9,11 @@
 
 from muni_portal.core.api import api_router
 from muni_portal.core.views import (
-    IndexView,
     CollaboratorWebhookApiView,
 )
 from muni_portal.core.views.api.service_requests import ServiceRequestDetailView, ServiceRequestListCreateView
 
 urlpatterns = [
-    path("", IndexView.as_view(), name="index"),
```

Comment:
Do we really not want a root/index page even if blank? It will 404 if the app is visited.